integration/src/test/resources/integration/homepage_to_competencypage.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Viewing the competency page
+  Scenario: Employee can click from the homepage to the competency page
+    Given as a kainos employee I want to get to the competencies per band page from the home page
+    When I click on the bands button
+    When I click on the <band> heading
+    Then I should be on the competencies page for that band
```

Comment:
The wording here gives a false impression of specificity. It reads like it's verifying you're on the `Apprentice` competencies page, or the `Trainee` competencies page – but all you're actually verifying in both cases is that it is **a** competencies page.